src/cursor.rs

```diff
@@ -376,18 +208,15 @@ impl Command for Hide {
 
     #[cfg(windows)]
     fn execute_winapi(&self) -> Result<()> {
-        WinApiCursor::new().hide()
+        sys::show_cursor(false)
     }
 }
 
-/// A command to show the cursor.
+/// A command that shows the terminal cursor indicator.
```

Comment:
Can we remove the `indicator` word? We don't use it anywhere in the docs. It was added in this PR for just two commands `Hide` & `Show`.